packages/sanity/src/core/studio/components/navbar/search/components/filters/filter/inputs/date/datePicker/calendar/CalendarMonth.tsx

```diff
@@ -1,50 +1,37 @@
 import {Box, Grid, Text} from '@sanity/ui'
-import {isSameDay, isSameMonth} from 'date-fns'
 import React from 'react'
-import {useDatePicker} from '../contexts/useDatePicker'
+import {useCalendar} from './contexts/useDatePicker'
 import {CalendarDay} from './CalendarDay'
 import {WEEK_DAY_NAMES} from './constants'
 import {getWeeksOfMonth} from './utils'
 
 interface CalendarMonthProps {
-  date: Date
-  focused?: Date
-  selected?: Date
-  onSelect: (date: Date) => void
   hidden?: boolean
+  onSelect: (date: Date) => void
 }
 
-export function CalendarMonth(props: CalendarMonthProps) {
-  const {fontSize} = useDatePicker()
+export function CalendarMonth({hidden, onSelect}: CalendarMonthProps) {
+  const {focusedDate, fontSize} = useCalendar()
 
   return (
-    <Box aria-hidden={props.hidden || false} data-ui="CalendarMonth">
-      <Grid gap={1} style={{gridTemplateColumns: 'repeat(7, minmax(44px, 46px))'}}>
+    <Box aria-hidden={hidden || false} data-ui="CalendarMonth">
+      <Grid style={{gridTemplateColumns: 'repeat(7, minmax(44px, auto))'}}>
```

Comment:
Since the inline styling is a static object, I would prefer moving it to a `styled(Grid)` component!